worlds/oot_soh/Options.py

```diff
@@ -204,27 +204,24 @@ class TriforceHunt(Toggle):
     display_name = "Triforce Hunt"
 
 
-class TriforceHuntRequiredPieces(Range):
+class TriforceHuntPiecesTotal(Range):
     """
-    The amount of Triforce pieces required to win the game.
+    Specify an exact number of Triforce Pieces to add to the item pool. If the item pool is out of space, no more will be added.
     """
-    display_name = "Triforce Hunt Required Pieces"
+    display_name = "Triforce Hunt Pieces Total"
     range_start = 1
     range_end = 100
-    default = 20
+    default = 30
 
 
-class TriforceHuntExtraPiecesPercentage(Range):
+class TriforceHuntPiecesRequired(Range):
     """
```

Comment:
```suggestion
class TriforceHuntPiecesRequiredPercentage(Range):
    """
```
And also rename all the relevant other parts where this option is used etc.

Reply:
Manually applying this so I can change it everywhere.